README.md

```diff
@@ -1,5 +1,95 @@
 # PayTonight: Payment Gateway: 
 
-It is now available at: https://paytonight.herokuapp.com/
+## Documentation
 
-### Documentation SOON
+## How to use
+
+  1. Register in API through Portal https://paytonight.herokuapp.com/
+
+  2. Login and generate API credentials, make sure to save APP_KEY or you will need to generate another one (notice that APP ID doesn't change so)
+
+  3. Make a POST Call to /api/payment/checkout with the following headers:
+  ```
+    Content-Type: 'application/json',
+    x-api-key: "<your API key>"
+  ```
+  and the following body:
+
+  ```
+    APP_ID:"<Your APP_ID>"
+    amount: <Amount charged>
+  ```
+  4. A payment frame should return
```

Comment:
The page will be redirected to `/api/payment/iframe?token=token` with the following iframe
![payment-iframe](./public/assets/img/iframe.png)